server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/MailboxSetMethod.scala

```diff
@@ -38,16 +38,13 @@ import reactor.core.scheduler.Schedulers
 
 import scala.collection.immutable
 
+case class MailboxHasMail(mailboxId: MailboxId) extends Exception
+case class MailboxHasChild(mailboxId: MailboxId) extends Exception
```

Comment:
May be we should rename these to MailboxHasMailException & MailboxHasChildException